packages/teleport-monitoring/.env.example

```diff
@@ -1,2 +1,3 @@
 DATABASE_URL=postgresql://postgres:password@localhost:5432/local
 L1_RPC=E2E test require goerli rpc
+CONFIG='{"slaves": [{"name":"OPT-GOER-A", "l2Rpc":"https://goerli.optimism.io/"}]}'
```

Comment:
I can see that by default this monitor has two domains `ARB-GOER-A` and `OPT-GOER-A`. Does the user have to provide both of these or if they provide just one it'll lose the other one?

Reply:
Providing just one is fine, like in this example. If the other l2Rpc is not specified, it will default to its value in `src/config.ts`.